### PR TITLE
Create standalone OAuth classes

### PR DESCRIPTION
**Pull request by yangkcatiu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,55 +103,32 @@ api_key_secret = os.environ.get("TWITTER_API_KEY_SECRET")
...
-> **NOTE:** There are two different types of Oauth methods. [OAuth 1.0a](https://developer.twitter.com/en/docs/authentication/oauth-1-0a) (User Context) and [OAuth 2.0 Bearer Token](https://developer.twitter.com/en/docs/authentication/oauth-2-0) (App Context). **`osometweet` utilizes the OAuth 2.0 Bearer Token by default. This matters because, for certain endpoints, how many requests you can make during a given period of time changes based on which type of authorization you are using**. The best place to see these comparisons is under the _Migrate_ pages for a given endpoint in Twitter's documentation. Here is an example for the [User Lookup endpoints](https://developer.twitter.com/en/docs/twitter-api/users/lookup/migrate).
 
-### Controlling which authorization type you'd like to use
-As mentioned above, `osometweet` defaults to OAuth 2.0 Bearer Token authorization. If you'd like to use OAuth 1.0a authorization you can do that in two ways. 
+### OAuth 2
+OAuth 2 only requires you bearer token. You can create an `OAuth2` object with the folliwng code:
```
Comment (mr-devs) [inline README.md:108]: "you" should be "your"

File: tests/tests.py [hunk 2]
```diff
@@ -14,28 +14,49 @@ class TestOauth(unittest.TestCase):
     Make sure the oatuh is working
```
Comment (mr-devs) [inline tests/tests.py:14]: "oatuh" should be "oauth"

**Comment by mr-devs** [comment]
This looks great! Thanks for making these changes. I added super minor notes on some of the comments. I wasn't sure if I am allowed to make edits to your pull request so I hope this isn't annoying. 😄

**Comment by mr-devs** [comment]
I'm just going to merge and then make these changes myself. :)

**Comment by yangkcatiu** [comment]
Thanks!

**Pull request merged**